database/seed.js

```diff
@@ -59,14 +59,18 @@ const itemData = [
 const generateRecords = (num) => {
   var itemId = 110;
 
-  const digits = [0,1,2,3,4,5,6,7,8,9];
+  //const digits = [0,1,2,3,4,5,6,7,8,9];
 
   for (var i = 0; i < num; i++) {
-    var newDoc;
+    var lastDigit = Math.floor(Math.random() * 10);
 
-    newDoc = {
+    if (lastDigit === 0) {
+      lastDigit = 9;
```

Comment:
@margotTenenbaum Why do we need to reassign the last digit to 9 in this case?

Reply:
Hi @GrumpyCat0709,
If the last digit is a 0, type coercion cuts off that digit. So 9.90 becomes 9.9
I tried a half dozen different ways of preserving the 0 but couldn't get anything to work, so decided to prevent the price from ever ending in a 0